vm/src/stdlib/builtins.rs

```diff
@@ -367,7 +365,13 @@ mod builtins {
         obj.hash(vm)
     }
 
-    // builtin_help
+    #[pyfunction(name = "breakpoint")]
+    fn builtin_breakpoint(args: FuncArgs, vm: &VirtualMachine) -> PyResult {
```

Comment:
```suggestion
    #[pyfunction]
    fn breakpoint(args: FuncArgs, vm: &VirtualMachine) -> PyResult {
```
if name doesn't conflict, we can use the python function name here